src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1383,7 +1421,7 @@ private static void ForceUpdateOfAllHttpClients()
             }
         }
 
-        private static bool OnServerCertificateCustomValidationCallback(HttpRequestMessage msg, X509Certificate2 cert, X509Chain chain, SslPolicyErrors errors)
+        public static bool OnServerCertificateCustomValidationCallback(HttpRequestMessage msg, X509Certificate2 cert, X509Chain chain, SslPolicyErrors errors)
```

Comment:
can be made internal?

Reply:
we need to change signature to the one matching RemoteValidationCallback I think?

Reply:
We can but then we'll need to write a second translator to use in CreateHttpMessageHandler() where we assign the callback to the httpMessageHandler